eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala

```diff
@@ -53,13 +54,11 @@ object ChannelRelayer {
     case _ => Map.empty
   }
 
-  private type ChannelUpdates = Map[ShortChannelId, Relayer.OutgoingChannel]
-  private type NodeChannels = mutable.MultiDict[PublicKey, ShortChannelId]
-
   def apply(nodeParams: NodeParams,
             register: ActorRef,
-            channelUpdates: ChannelUpdates = Map.empty,
-            node2channels: NodeChannels = mutable.MultiDict.empty[PublicKey, ShortChannelId]): Behavior[Command] =
+            channels: Map[ByteVector32, Relayer.OutgoingChannel] = Map.empty,
+            scid2channels: Map[ShortChannelId, ByteVector32] = Map.empty,
```

Comment:
nit: maybe call this `resolveScid` to be consistent with the router?

Reply:
I'd rather rename `resolveScid` to `scid2channels` 😆

Reply:
True!